app/models/admin_auction_status_presenter_factory.rb

```diff
@@ -7,18 +7,16 @@ def initialize(auction:)
 
   def create
     if auction.payment_confirmed? || auction.c2_paid?
-      Object.const_get("C2StatusPresenter::#{c2_status}").new(auction: auction)
+      Object.const_get("C2StatusPresenter::#{c2_status}")
     elsif future? && auction.published?
-      AdminAuctionStatusPresenter::Future.new(auction: auction)
+      AdminAuctionStatusPresenter::Future
     elsif work_in_progress?
-      AdminAuctionStatusPresenter::WorkInProgress.new(auction: auction)
-    elsif auction.accepted? && auction.accepted_at.nil?
-      AdminAuctionStatusPresenter::PendingPaymentUrl.new(auction: auction)
+      AdminAuctionStatusPresenter::WorkInProgress
     elsif !auction.pending_delivery?
-      Object.const_get("AdminAuctionStatusPresenter::#{status}").new(auction: auction)
+      Object.const_get("AdminAuctionStatusPresenter::#{status}")
     else # if auction.purchase_card == 'default'
-      Object.const_get("C2StatusPresenter::#{c2_status}").new(auction: auction)
-    end
+      Object.const_get("C2StatusPresenter::#{c2_status}")
+    end.new(auction: auction)
```

Comment:
This is consistent with BidStatusPresenter, so I approve.

My only complaint about this approach is that it forces you to have the same arguments for all presenters even if they don't use them. For instance, when I was doing the BidStatusPresenter::Available::Vendor::BidError presenter, I really didn't need the user but I did need an bid error. To make it work, I had to have it so all auction presenters accepted a bid_error argument, even though none of the other ones used them. It seems like this could be an issue down the road, not enough to affect this PR, but I wanted to note it.